src/composables/useLocalStorage.js

```diff
@@ -0,0 +1,58 @@
+import { ref } from 'vue'
+
+/**
+ * @name addToLocalStorage
+ * @function
+ * @param {String} item Item to add to local storage
+*/
+
+/**
+ * @name removeFromLocalStorage
+ * @function
+ * @param {String} item Item to remove from local storage
+*/
+
+/**
+ * @typedef {Object} LocalStorage
+ * @property {string} itemOnLoad - The default item on load
+ * @property {ref} item - The selected item
+ * @property {ref} items - The array of items in local storage
```

Comment:
This wasn't quite working with VSCode's type detection features, but I managed to figure out the right syntax.

However, personally I find JSDoc a bit fiddly to work with and if we want this level of type definition we are probably better off adopting TypeScript in future.

```suggestion
 * @callback addToLocalStorage
 * @param {*} item Item to add to local storage
 * @return {void}
*/

/**
 * @callback removeFromLocalStorage
 * @param {*} item Item to remove from local storage
 * @return {void}
*/

/**
 * @typedef {Object} LocalStorage
 * @property {*} itemOnLoad - The default item on load
 * @property {import('vue').Ref} item - The selected item
 * @property {import('vue').Ref<Set>} items - The array of items in local storage
```